templates/web/base/contact/blurb.html

```diff
@@ -3,7 +3,10 @@
 </p>
 
 <p>
-[% loc('Please do <strong>not</strong> report problems through this form; messages go to
-the team behind this site, not a council. To report a problem,
-please <a href="/">go to the front page</a> and follow the instructions.') %]
+  [% TRY %]
+    [% INCLUDE 'contact/_blurb.html' %]
```

Comment:
I wonder if it might be clearer to simply override the whole `contact/blurb.html` template for Peabody, rather than introducing another partial?